sigs.yaml

```diff
@@ -3248,6 +3248,44 @@ workinggroups:
     liaison:
       github: cblecker
       name: Christoph Blecker
+- dir: wg-lts
+  name: LTS
+  mission_statement: >
+    Define LTS
+
+  stakeholder_sigs:
```

Comment:
As @pacoxu already mentioned, let's put sig node here as well for the kubelet skew

Reply:
Whether or not kubelet skew is relevant depends on the recommended changes, which are not yet known. I think ~every sig could be included as a stakeholder here, but I think it makes more sense to scope the sponsoring sigs to the ones impacted regardless of changes recommended

Reply:
/lgtm